.github/workflows/main.yml

```diff
@@ -12,7 +12,7 @@ jobs:
         os:
         - macos-latest
         - ubuntu-latest
-        - windows-latest
+        #- windows-latest # FIXME
```

Comment:
Hey @twpayne, I just happened to come across this tonight and I wanted to let you know I'm available to help test or fix stuff on Windows if needed.

Reply:
Thanks @zb140, that's really appreciated :) I will definitely reach out to you.